packages/connectivity/src/scp-cf/destination/destination.ts

```diff
@@ -240,7 +240,13 @@ function setDefaultAuthenticationFallback(
     : assoc('authentication', getAuthenticationType(destination), destination);
 }
 
-function parseCertificate(
+/**
+ * Transforms the lower case variables of the destination response to lower case.
+ * @internal
+ * @param certificate
+ * @returns The parsed destination certificate.
+ */
```

Comment:
This seems incomplete and like a copy paste error.

Reply:
I just checked and the analogous method for destination is also called parseDestiantion